src/Data/Integer/Properties.agda

```diff
@@ -296,6 +292,32 @@ distribʳ-⊖-+-neg a b c = begin
   rewrite ℕₚ.+-assoc (suc a) (suc b) (suc c)
         = refl
 
+suc-pred : ∀ m → sucℤ (pred m) ≡ m
+suc-pred m = begin
+  sucℤ (pred m) ≡⟨ sym (+-assoc (+ 1) (- + 1) m) ⟩
+  + 0 + m       ≡⟨ +-identityˡ m ⟩
+  m ∎ where open ≡-Reasoning
+
+pred-suc : ∀ m → pred (sucℤ m) ≡ m
+pred-suc m = begin
+  pred (sucℤ m) ≡⟨ sym (+-assoc (- + 1) (+ 1) m) ⟩
+  + 0 + m       ≡⟨ +-identityˡ m ⟩
+  m ∎ where open ≡-Reasoning
+
++-pred : ∀ m n → m + pred n ≡ pred (m + n)
```

Comment:
Hmm the naming and the direction of the equality of these two should probably agree with `neg-distribˡ-*` earlier in the file?

Reply:
This one follows [`+-suc`](https://agda.github.io/agda-stdlib/Data.Nat.Properties.html#9415).